deploy-cluster-aws/example_deploy_conf.py

```diff
@@ -49,3 +49,19 @@
 # Examples: "m3.2xlarge", "c3.8xlarge", "c4.8xlarge"
 # For all options, see https://aws.amazon.com/ec2/instance-types/
 INSTANCE_TYPE="m3.2xlarge"
+
+# USING_EBS is True if you want to attach an Amazon EBS volume
+USING_EBS=False
+
+# EBS_VOLUME_SIZE is the size of the EBS volume to attach, in GiB
+# Since we assume 'gp2' volumes (for now), the possible range is 1 to 16384
+# If USING_EBS=False, EBS_VOLUME_SIZE is irrelevant and not used
+EBS_VOLUME_SIZE=30
+
+# EBS_OPTIMIZED is True or False, depending on whether you want
+# EBS-optimized instances. See:
+# https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSOptimized.html
+# Not all instance types support EBS optimization.
+# Setting EBS_OPTIMIZED=True may cost more, but not always.
+# If USING_EBS=False, EBS_OPTIMIZED is irrelevant and not used
+EBS_OPTIMIZED=False
```

Comment:
What happens if I set EBS_OPTIMIZED to `true` and select an INSTANCE_TYPE that does not support it? will it just be ignored?